carma-messenger-core/truck_inspection_plugin/README.md

```diff
@@ -0,0 +1 @@
+# truck_inspection_plugin
```

Comment:
Add link to design doc here
https://usdot-carma.atlassian.net/wiki/spaces/CRMPLT/pages/1319928028/Detailed+Design+-+Traffic+Incident+Node